src/mongocxx/options/find.cpp

```diff
@@ -42,12 +42,12 @@ void find::limit(std::int32_t limit) {
     _limit = limit;
 }
 
-void find::max_await_time_ms(std::int64_t max_await_time_ms) {
-    _max_await_time_ms = max_await_time_ms;
+void find::max_await_time(std::chrono::milliseconds max_await_time) {
+    _max_await_time = max_await_time;
```

Comment:
std::move, here and below. I'll stop mentioning it here, but please check the other setters